plugins/openvr/src/OpenVrHelpers.cpp

```diff
@@ -108,9 +108,11 @@ QString getVrSettingString(const char* section, const char* setting) {
     return result;
 }
 
+int headsetNotConnected = -1;
```

Comment:
Please use a `bool`     and use the boolean way of writing a variable name see [CODING_STANDARD.md#1.2.10]( https://github.com/kasenvr/project-athena/blob/kasen/core/CODING_STANDARD.md#1210-the-prefix-is-should-be-used-for-boolean-variables-and-methods)  in this case it would be something like `bool isHMDConnected = true`

Reply:
oops I forgot to add something that actually used it.... I guess it doesn't even need that now as it actually works